presto-tests/src/main/java/com/facebook/presto/tests/tpch/ConcatRecordSet.java

```diff
@@ -157,6 +157,12 @@ public double getDouble(int field)
         }
 
         @Override
+        public Object getObject(int field)
+        {
+            throw new UnsupportedOperationException();
```

Comment:
But TPCH doesn't have structural types. Implementing it will be adding dead code.